Makefile

```diff
@@ -0,0 +1,72 @@
+SHELL := /usr/bin/env bash
+
+# build the registry image
+IMAGE_REGISTRY?=quay.io
+IMAGE_REPOSITORY?=openshift-sre
+IMAGE_NAME?=osd-operators
+CATALOG_NAMESPACE?=openshift-operator-lifecycle-manager
+DOCKERFILE?=./Dockerfile
+CHANNEL?=production
+GIT_SHA=$(shell git rev-parse HEAD | cut -c1-8)
+
+TEMPLATE_CS=templates/template_osd-operators.CatalogSource.yaml
+DEST_CS=manifests/00_osd-operators.CatalogSource.yaml
+
+SUBSCRIPTIONS=$(shell cat subscriptions.json)
+
+.PHONY: default
+default: build
+
+.PHONY: clean
+clean:
+	rm -rf manifests/
+
+.PHONY: catalogSource
+catalogSource:
+	# create CatalogSource yaml
+	mkdir -p manifests/
+	cp ${TEMPLATE_CS} ${DEST_CS}
+	sed -i "s/#IMAGE_REGISTRY#/${IMAGE_REGISTRY}/g" ${DEST_CS}
+	sed -i "s/#IMAGE_REPOSITORY#/${IMAGE_REPOSITORY}/g" ${DEST_CS}
+	sed -i "s/#IMAGE_NAME#/${IMAGE_NAME}/g" ${DEST_CS}
+	sed -i "s/#CATALOG_NAMESPACE#/${CATALOG_NAMESPACE}/g" ${DEST_CS}
+	sed -i "s/#CHANNEL#/${CHANNEL}/g" ${DEST_CS}
+	sed -i "s/#GIT_SHA#/${GIT_SHA}/g" ${DEST_CS}
+
+.PHONY: operatorManifests
+operatorManifests:
+	# create Subscription yaml (many)
+	mkdir -p manifests/
+	PSN=0; while true; do \
+		OPERATOR=`cat operators.json | jq -r .[$${PSN}]`; \
```

Comment:
Minor style nit: `$(sub command)` is a little more readable, in that backticks are more likely to be confused for `'`